js/build-advanced-directory.js

```diff
@@ -1137,16 +1137,19 @@ AdvancedDirectory.prototype.directoryNotConfigured = function(){
 };
 
 AdvancedDirectory.prototype.flViewportRedraw = function(){
```

Comment:
is `flViewportRedraw` supposed to be chained?

Reply:
We haven't used it in a chained way, but I changed it in the standard directory so that it is chained. The chaining is used in one instance.

I'm therefore bringing the advanced directory to use the same structure.